cmd/json-ordered-tidy/main.go

```
@@ -72,24 +110,91 @@ func usage(err string) {
   order. If given a directory it looks for files matching the given -ext
   value.
 
+  This tidier is capable of sorting object keys in arbitrary orders, as well
+  as optional sorting the contents of arrays. You can configure this using a
+  JSON-based config file.
+
+  The config file should be a JSON object. It can contain three keys,
+  "indent", "keyOrder" and "arraySort". You can specify just one key as
+  well. Note that specifying "indent" in the config file will override any
+  command line.
```

Personally, I'd silently discarding a command line arg in favor of a config value surprising behavior. I'd suggest either allowing the command line arg to take precedence or printing a warning if it's specified in both places.

---

It now warns if you provide both _and_ they are different. It tells you it'll use the one in the config file.